treeherder/push_health/tests.py

```diff
@@ -124,6 +124,14 @@ def get_current_test_failures(push, option_map, jobs):
             job['failure_classification_id'] == 4 for job in jobs[job_name]
         )
 
+        isInvestigated = False
+        investigatedTestId = None
+        for investigatedTest in investigatedTests:
+            if investigatedTest.test == test_name:
```

Comment:
This should also be checking that the job_type_id matches.  Currently, it marks all failures on all platforms for this test as investigated, when only one was checked to be marked.